staff/dtiburcio/JS-Exercises/JavascriptExercises.js

```diff
@@ -0,0 +1,177 @@
+    var result = 0;
+
+   for (var i=0; i <= quantity; i++){
+        result += arraySum[i];
+    }
+
+    return result;
+}
+
+function multiply(arrayToMultiply){
+    var arrayMultiply = arrayToMultiply;
+    var quantity = arraySum.length - 1;
+    var result = 1;
+
+   for (var i=0; i <= quantity; i++){
+        result *= arraySum[i];
+    }
+
+    return result;
+}
+
+/*
+reverse()
+Define a function reverse() that computes the reversal of a string. For example, reverse("jag testar") should return the string "ratset gaj".
+*/
+
+function reverse(stringToReverse){
+    var stringReverse = stringToReverse;
+    var arrayResult = [];
+
+    for (var i = 0; i < stringReverse.length; i++){
+        arrayResult[stringReverse.length - i] = stringReverse[i];
+    }
+
+    result = arrayResult.toString().replace(/,/g, '');
+    return result;
+}
+
+/*
+translate()
+Represent a small bilingual lexicon as a Javascript object in the following fashion {"merry":"god", "christmas":"jul", "and":"och", "happy":gott", "new":"nytt", "year":"år"} and use it to translate your Christmas cards from English into Swedish.
+*/
+
+function translate(stringtoTranslate){
+    var stringTranslate = stringtoTranslate.split(' ');
+    var words = stringTranslate.length - 1;
+
+    for (var i = 0; i <= words; i++){
+        switch (stringTranslate[i]) {
+            case 'merry': 
+                stringTranslate[i] = 'god'; 
+                break;
+            case 'christmas':
+                stringTranslate[i] = 'jul';
+                break;
+            case 'and':
+                stringTranslate[i] = 'och';
+                break;
+            case 'happy':
+                stringTranslate[i] = 'gott';
+                break;
+            case 'new':
+                stringTranslate[i] = 'nytt';
+                break;
+            case 'year':
+                stringTranslate[i] = 'år';
+                break;
+            default:
+                break;
+          }
+    }
+    var result = stringTranslate.join();
+    var result2 = result.replace(/,/g,' ');
+    return result2;
+}
+
+/*
+findLongestWord()
+Write a function findLongestWord() that takes an array of words and returns the length of the longest one.
+*/
+
+function findLongestWord(stringToEvaluate){
+    var stringEvaluate = stringToEvaluate.split(' ');
+    var words = stringEvaluate.length - 1;
+    var result = '';
+    var max = 0;
+
+    for (var i = 0; i <= words; i++){
+        temp = stringEvaluate[i].length;
+        
+        if (temp > max){
+            result = stringEvaluate[i];
+            max = temp;
+        }
+    }
+    return max;
+}
+
+/*
+filterLongWords()
+Write a function filterLongWords() that takes an array of words and an integer i and returns the array of words that are longer than i.
+*/
+
+function filterLongWords(stringOfWords, mininum){
+    var listOfWords = stringOfWords.split(' ');
+    var words = listOfWords.length - 1;
+    var min = mininum;
```

Comment:
You can just use the name of the argument in your logic

Reply:
Ok. Changed.